internal/engine/runtimelog/podlogmanager_test.go

```diff
@@ -250,7 +258,7 @@ func TestLogReconnection(t *testing.T) {
 
 	_, _ = writer.Write([]byte("hello world!"))
 	f.AssertOutputContains("hello world!")
-	assert.Equal(t, startTime, f.kClient.LastPodLogStartTime)
+	assert.Equal(t, startTime.Truncate(time.Second), f.kClient.LastPodLogStartTime)
```

Comment:
Just curious - is this result of (intentionally) reduced precision or something else?

Reply:
ya, this is not documented super-well, but metav1.Time only provides second-level granularity https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Time.Rfc3339Copy
so all the underlying Log apis truncate this anyway

this test changed because we're serializing the API, which is where the truncation happens